proof/infoflow/ARM/ArchRetype_IF.thy

```diff
@@ -12,6 +12,16 @@ context Arch begin global_naming ARM
 
 named_theorems Retype_IF_assms
 
+lemma do_ipc_transfer_valid_arch_no_caps[wp]:
+      "\<lbrace>valid_arch_state \<rbrace> do_ipc_transfer s ep bg grt r
+      \<lbrace>\<lambda>rv. valid_arch_state\<rbrace>"
```

Comment:
Indentation here and in following lemma (and in RISCV64)

Reply:
it was copy-pasta, but indeed atrocious, cleaned up to:
```
"do_ipc_transfer s ep bg grt r ⦃valid_arch_state⦄"
```